shared/login/register/set-public-name/index.render.desktop.js

```diff
@@ -4,27 +4,30 @@ import {Text, Button, Input, Icon} from '../../../common-adapters'
 import Container from '../../forms/container'
 import type {Props} from './index.render'
 
-const SetPublicName = ({onBack, onSubmit, onChange, deviceNameError, deviceName, waiting, submitEnabled = true}: Props) => (
-  <Container style={stylesContainer} onBack={onBack}>
-    <Text type='Header' style={stylesHeader}>Set a public name for this device:</Text>
-    <Icon type='icon-computer-colors-64' style={stylesIcon} />
-    <Input
-      autoFocus={true}
-      errorText={deviceNameError}
-      style={stylesInput}
-      floatingLabelText='Device name'
-      onEnterKeyDown={() => onSubmit()}
-      onChange={event => onChange(event.target.value)}
-      value={deviceName} />
-    <Button
-      type='Primary'
-      style={stylesButton}
-      disabled={!submitEnabled}
-      waiting={waiting}
-      label='Continue'
-      onClick={() => onSubmit()} />
-  </Container>
-)
+const SetPublicName = ({onBack, onSubmit, onChange, deviceNameError, deviceName, waiting, submitEnabled}: Props) => {
```

Comment:
we should make submitEnabled always a boolean and disallow null. it can be inited to false in the store
also we could just use the default pararms also right (it does an explicit check against undefined but, ...,waiting, submitEnabled = true}: Props